### PR TITLE
test(feature-card): added e2e for medium and large stories

### PR DESCRIPTION
**Pull request by IgnacioBecerra**
### Related Ticket(s)
#7435
#7436

### Description
This PR introduces the e2e tests for the Feature Card component, for both the medium and large variations.

Note: For some reason, trying to use `click()` on the FeatureCard (even on the actual `a` element) makes the test hang indefinitely. When dispatching an event or even using `trigger('click')` on the element still doesn't work, yet.

### Changelog

**New**

- e2e Feature Card tests for both React and Web Components

### CODE REVIEW

File: packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js [hunk 1]
```diff
@@ -0,0 +1,465 @@
...
+      // TODO: click states currently not working in percy for web components
+      // cy.percySnapshot('FeatureCard medium | g100 theme', {
+      //   widths: [1280],
+      // });
+    });
+  });
+});
+
+describe('FeatureCard medium (mobile)', () => {
+  beforeEach(() => {
```
Comment (annawen1) [inline packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js:135]: Looking good, I think there's no need to copy over the same tests for mobile. Only thing I might keep is checking that the click functionality on the entire card still works when in mobile

File: packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+      const positionValue = after.getPropertyValue('position');
+      const insetValue = after.getPropertyValue('inset');
+
+      expect(positionValue).to.eq('absolute');
+      expect(insetValue).to.eq('0px');
+    });
+
+    cy.screenshot();
+    // Take a snapshot for visual diffing
+    // TODO: click states currently not working in percy for web components
```
Comment (ariellalgilmore) [inline packages/react/tests/e2e-storybook/cypress/integration/FeatureCard/FeatureCard.e2e.js:58]: Looks good!! only small thing is changing the comments for react if there any percy snapshots we want there

**Review by ariellalgilmore — APPROVED** [review]
LGTM!

**Review by annawen1 — APPROVED** [review]
LGTM

**Pull request merged**